### PR TITLE
Use yarn workspaces

### PR DESCRIPTION
**Pull request by jfinkhaeuser**
Implements #45

### CODE REVIEW

**Comment by jfinkhaeuser** [comment]
This is a huge diff that doesn't actually add any code. In summary:
- Move code into different packages
- Remove any synchronization/protocol code.

But by all means, review it thoroughly @mnaamani  :)

**Comment by mnaamani** [comment]
Good thing I anticipated such a PR and added an [exception](https://github.com/Joystream/joystream/pull/43/files#diff-04c6e90faac2675aa89e2176d2eec7d8R88) ;)

But I will go through it non the less

File: packages/colossus/5CpGKFTCdub9TWVD8bpR1kZbwvBrWXJ53idqYNUSuk8hS5hJ.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"address":"5CpGKFTCdub9TWVD8bpR1kZbwvBrWXJ53idqYNUSuk8hS5hJ","encoded":"0x3053020101300506032b65700422042058cc8db354ab43e4c318d641c3a8d78b451ff01bf15cf6beea53fa88ec0b40fd2138d116e4cb9e41b9c2012b3d0db4158c06e182cec68f4c5245c8b0f36b5558a1230321002138d116e4cb9e41b9c2012b3d0db4158c06e182cec68f4c5245c8b0f36b5558","encoding":{"content":["pkcs8","ed25519"],"type":"none","version":"2"},"meta":{"name":"storage role account for 5HfS9kwKMUxVpW5JoKnqvHE7tTz32wDN59p3BWRmYykygMK7"}}
```
[the remark this answers is not in the document]
Comment (jfinkhaeuser) [reply packages/colossus/5CpGKFTCdub9TWVD8bpR1kZbwvBrWXJ53idqYNUSuk8hS5hJ.json]: Oh yes - good catch!

File: packages/colossus/foo/id [hunk 2]
```diff
@@ -0,0 +1 @@
+af42003e-955c-4a22-b1c3-1d8c7876645b
```
[the remark this answers is not in the document]
Comment (jfinkhaeuser) [reply packages/colossus/foo/id]: No... I have no idea why I added this.

**Comment by jfinkhaeuser** [comment]
> The `devel` branch should be reamed to `development` (This can be done after merge obviously)
Hmm, fair enough. I hadn't read the guidelines at this point. No worries about that :)

**Comment by jfinkhaeuser** [comment]
> Can we try to have all tests passing, or disable/remove tests that are no longer relevant.

Yes... ish.

The reason I didn't modify the tests too much is because the failing ones pertain to hyperdrive, which we want to remove. The current repo allows running a non-synchronizing node that acts as before the PR. I would prefer removing the hyperdrive related things including the failing tests in relation to the #65 issue.

Similarly, most files fail the linting at this point in time - I'd prefer to fix that in a separate PR that just fixes things reported by the linter.

**Comment by jfinkhaeuser** [comment]
Also, following this PR, I'd like to add a separate PR for #64

**Pull request merged**